app/forms/prior_authority/steps/check_answers_form.rb

```diff
@@ -23,11 +23,8 @@ def application_corrected
       end
 
       def application_changed_since_request?
-        last_updated_at > application.resubmission_requested
-      end
-
-      def last_updated_at
-        LastUpdateFinder.call(application)
+        content = SubmitToAppStore::PriorAuthorityPayloadBuilder.new(application:).data
```

Comment:
Oh, something to look out for - at the moment EventBuilder is calling `deep_stringify_keys` on this hash in its initializer, before passing it in the ChangeLister. You might find that without doing that here, ChangeLister produces unexpected results. So might be work doing a deep_stringify_keys _inside_ change_lister so you don't need to call it before passing the data in.